src/main/java/ua/ithillet/lesson6/WorkWithListCollection.java

```diff
@@ -0,0 +1,62 @@
+package ua.ithillet.lesson6;
+
+import java.util.*;
+
+public class WorkWithListCollection {
+
+    public static int countOccurrence(List<String> list, String searchItem) {
+        int count = 0;
+        for (String item : list) {
+            if (item.equals(searchItem)) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    public static List<Integer> toList(Integer[] list) {
+        List<Integer> targetList = new ArrayList<>(List.of(list));
```

Comment:
```suggestion
        return List.of(list);
```
Може і так працювати (якщо немає потреби потім змінювати лист)